File: dungeons-26.1.2/src/main/java/fishmod/utils/rendering/RenderingEvent.java
package fishmod.utils.rendering;

import com.mojang.blaze3d.vertex.PoseStack;
import com.mojang.blaze3d.vertex.VertexConsumer;
import net.fabricmc.fabric.api.client.rendering.v1.level.LevelRenderContext;

public interface RenderingEvent {
    void render(LevelRenderContext context, PoseStack matrixStack, VertexConsumer consumer);

}

File: dungeons-26.1.2/src/main/kotlin/fishmod/utils/rendering/RenderingEvent.kt
package fishmod.utils.rendering

import com.mojang.blaze3d.vertex.PoseStack
import com.mojang.blaze3d.vertex.VertexConsumer
import net.fabricmc.fabric.api.client.rendering.v1.level.LevelRenderContext

fun interface RenderingEvent {
    fun render(context: LevelRenderContext, matrixStack: PoseStack, consumer: VertexConsumer)
}
